src/olympia/lib/akismet/models.py

```diff
@@ -0,0 +1,111 @@
+import requests
+
+from django.conf import settings
+from django.db import models
+
+import olympia.core.logger
+from olympia.amo.models import ModelBase
+
+
+log = olympia.core.logger.getLogger('z.lib.akismet')
+
+
+class AkismetReport(ModelBase):
+    UNKNOWN = -1
+    HAM = 0
+    DEFINITE_SPAM = 1
+    MAYBE_SPAM = 2
+    RESULT_CHOICES = (
+        (UNKNOWN, 'Unknown'),
+        (HAM, 'Ham'),
+        (DEFINITE_SPAM, 'Definite Spam'),
+        (MAYBE_SPAM, 'Maybe Spam'))
+    HEADERS = {'User-Agent': 'Mozilla Addons/3.0'}
+
+    # The following should normally be set before comment_check()
+    comment_type = models.CharField(max_length=255)
+    user_ip = models.CharField(max_length=255)
+    user_agent = models.CharField(max_length=255)
+    referrer = models.CharField(max_length=255)
+    user_name = models.CharField(max_length=255)
+    user_email = models.CharField(max_length=255)
+    comment = models.TextField()
+    comment_modified = models.DateTimeField()
+    content_link = models.CharField(max_length=255)
+    content_modified = models.DateTimeField()
+
+    # Non-comment properties:
+    result = models.PositiveSmallIntegerField(
+        null=True, choices=RESULT_CHOICES)
+    reported = models.BooleanField(default=False)
+
+    # Fks to the supported types of content we support submitting
+    # Just Rating at first
+    rating_instance = models.ForeignKey(
+        'ratings.Rating', related_name='+', null=True,
+        on_delete=models.SET_NULL)
+
+    class Meta:
+        db_table = 'akismet_reports'
+
+    def _get_data(self):
+        data = {
+            'blog': settings.SITE_URL,
+            'user_ip': self.user_ip,
+            'user_agent': self.user_agent,
+            'referrer': self.referrer,
+            'permalink': self.content_link,
+            'comment_type': self.comment_type,
+            'comment_author': self.user_name,
+            'comment_author_email': self.user_email,
+            # 'comment_author_url': '',
```

Comment:
Commented out code should not be committed.

Reply:
Those were the other potential parameters that Akismet accepted and we might send in the future.  I'll remove them for now.

Reply:
https://github.com/mozilla/addons-server/pull/9127/commits/b7d0e9a4cf96609765e37e165b4a2694cd8919c6